example/fittemplate.md

```diff
@@ -0,0 +1,157 @@
+
+    cfg = [];
+    cfg.template.headshape      = polhemus;
+    cfg.checksize               = inf;
+    cfg.individual.headmodel    = template;
+    cfg                         = ft_interactiverealign(cfg);
+    template                    = ft_transform_geometry(cfg.m, template);
+
+## Removing unshared details between Polhemus and the template
+
+To create the individualized head model it is important that the scalp
+surface of the template head model only contains features that are also in the Polhemus head
+shape, and vice versa. We can use **[ft_defacemesh](/reference/ft_defacemesh)** to remove some features.
+
+So first we visualise both meshes:
+
+    ft_plot_mesh(template.bnd);  
+    ft_plot_mesh(polhemus);  
+
+{% include image src="/assets/img/example/fittemplate/template_surface_polhemus_surface_plot.png" %}
+
+As the template model and the Polhemus measurement have exclusive features, we will now remove these  
+
+    defaced_template      = template.bnd(1);
+    defaced_template.unit = template.unit
+
+    cfg              = [];
+    cfg.translate    = [-40 0 -50];
+    cfg.scale        = [200 200 200];
+    cfg.rotate       = [0 0 0];
+    defaced_template =  ft_defacemesh(cfg, defaced_template);
+
+    cfg              = [];
+    cfg.translate    = [-40 0 -50];
+    cfg.scale        = [200 200 200];
+    cfg.rotate       = [0 0 0];
+    defaced_polhemus =  ft_defacemesh(cfg, polhemus);
+
+Now we have another look at the surfaces
+
+    ft_plot_mesh(defaced_template);  
+    ft_plot_mesh(defaced_polhemus);  
+
+{% include image src="/assets/img/example/fittemplate/defaced_template_surface_polhemus_surface_plot.png" %}
+
+
+## Determining and applying of the transformation
+
+### Method 1: On the basis of spheres fitted to the head shapes
+
+For the first approach we will fit a sphere to the three fiducials and we will fit another sphere to the template head model. On the basis of difference of the center and scaling between the two spheres we can derive a transformation containing information of the translation and the global scaling.
+
+Fit a sphere to the MRI template
+
+    cfg=[];
+    cfg.method='singlesphere';
+    template_sphere = ft_prepare_headmodel(cfg, defaced_template);
+
+Fit a sphere to the polhemus headshape
+
+    cfg=[];
+    cfg.method = 'singlesphere';
+    polhemus_sphere = ft_prepare_headmodel(cfg, defaced_polhemus);
+
+Determine the global scaling and translation
+
+    scale = polhemus_sphere.r/template_sphere.r;
+
+    T2 = [1 0 0 template_sphere.o(1);
+          0 1 0 template_sphere.o(2);
+          0 0 1 template_sphere.o(3);
+          0 0 0 1                ];
+
+    T1 = [1 0 0 -template_sphere.o(1);
```

Comment:
why are you not translating back to the center of the sphere as fitted to the individual Polhemus head shape?

As I understand it the idea is
- shift the template center so that it is in [0,0,0]
- scale the template as if it were a sphere
- move the template center from [0,0,0] to the individual center

Reply:
Isn't this what happens in line 124?

Reply:
no, that requires `polhemus_sphere.o`

Reply:
Oh I see, so Currently I'm moving it back to the template center